### PR TITLE
Makefile myadd ex

### PR DESCRIPTION
**Pull request by garymlin**
Adding the myadd example to recitation notes B folder.  You can demo the Makefile by making with explicit/implicit rules; just comment the explicit rule and uncomment the implicit rule directly below it.

### CODE REVIEW

File: B-Makefiles/practice/myadd/Makefile [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+#
+# Instead, we rely on one of make’s implicit rules:
+#
+#     $(CC) $(LDFLAGS) <all-dependent-.o-files> $(LDLIBS)
+#
+# Also note that make assumes that main depends on main.o,
+# so we can omit it if we want to.
+
+main: main.o myadd.o
+#main: myadd.o
```
Comment (kevin1) [inline B-Makefiles/practice/myadd/Makefile]: Forgot to delete comment?

Comment (garymlin) [inline B-Makefiles/practice/myadd/Makefile]: I originally grabbed Jae's default Makefile with those comments and didn't take them out.  Do you think I should?  I'm pretty in the middle on this.

Comment (ohEmily) [inline B-Makefiles/practice/myadd/Makefile]: Ahh this is cool! Thanks Gary! I would take out Jae's comments, but maybe just put at the top like "This is Jae's makefile with the comments removed. Toggling which lines are commented demonstrates how the implicit and explicit rules in makefiles are equivalent."

Comment (garymlin) [inline B-Makefiles/practice/myadd/Makefile]: sounds good!  just updated that!

**Comment by ohEmily** [comment]
LGTM :+1:

**Pull request merged**